openshift/rhmap-nginx-proxy.json

```diff
@@ -0,0 +1,186 @@
+                    "deploymentconfig": "nginx-proxy"
+                },
+                "template": {
+                    "metadata": {
+                        "creationTimestamp": null,
+                        "labels": {
+                            "app": "nginx-proxy",
+                            "deploymentconfig": "nginx-proxy"
+                        },
+                        "annotations": {}
+                    },
+                    "spec": {
+                        "containers": [
+                            {
+                                "name": "nginx-proxy",
+                                "image": "rhmap/nginx-18-centos",
+                                "ports": [
+                                    {
+                                        "containerPort": 8080,
+                                        "protocol": "TCP"
+                                    }
+                                ],
+                                "env": [
+                                    {
+                                        "name": "BASE_HOST",
+                                        "value": "${BASE_HOST}"
+                                    }, {
+                                        "name": "RHMAP_PLATFORM_PATH",
+                                        "value": "${RHMAP_PLATFORM_PATH}"
+                                    }
+                                ],
+                                "resources": {},
+                                "terminationMessagePath": "/dev/termination-log",
+                                "imagePullPolicy": "Always"
+                            }
+                        ],
+                        "restartPolicy": "Always",
+                        "terminationGracePeriodSeconds": 30,
+                        "dnsPolicy": "ClusterFirst"
+                    }
+                }
+            },
+            "status": {
+                "observedGeneration": 2,
+                "replicas": 1,
+                "updatedReplicas": 1,
+                "unavailableReplicas": 1
+            }
+        },
+        {
+            "kind": "Route",
+            "apiVersion": "v1",
+            "metadata": {
+                "name": "proxy-route",
+                "creationTimestamp": null,
+                "labels": {
+                    "app": "nginx-proxy"
+                },
+                "annotations": {
+                    "openshift.io/host.generated": "true"
+                }
+            },
+            "spec": {
+                "host": "${EXTERNAL_ROUTE}",
+                "to": {
+                    "kind": "Service",
+                    "name": "nginx-proxy",
+                    "weight": 100
+                },
+                "port": {
+                    "targetPort": "8080-tcp"
+                },
+                "tls": {
+                    "termination": "edge",
+                    "insecureEdgeTerminationPolicy": "Redirect"
+                }
```

Comment:
It would update protocol from http to https. Access using http would not be possible. As we can imagine this may be accessed by another proxy so customers may update this part for their needs. I feel that this is sensible default.